includes/specials/SpecialUndelete.php

```diff
@@ -328,9 +328,9 @@ function isDeleted() {
 	function undelete( $timestamps, $comment = '', $fileVersions = array(), $unsuppress = false ) {
 		global $wgUser;
 
-		$resultMock = '';
-		if ( !$this->title->userCan( 'undelete' ) ) {
```

Comment:
Could you explain why the change from Title::userCan() to Title::getPermissionsErrors()? Could we not have used the hook in Title::userCan like we do with Wikia::canEditInterfaceWhitelist?